docs/input.rst

```diff
@@ -0,0 +1,269 @@
+    
+    :param uint64 inputHandle: The handle of the controller you want to bring up the binding screen for. Obtained from :func:`input.getConnectedControllers`.
+    :returns: (`boolean`) `true` for success; `false` if overlay is disabled/unavailable, or the user is not in Big Picture Mode.
+    :SteamWorks: `ShowBindingPanel <https://partner.steamgames.com/doc/api/isteaminput#ShowBindingPanel>`_
+
+.. function:: input.stopAnalogActionMomentum()
+    
+    :param uint64 inputHandle: The handle of the controller to affect. Obtained from :func:`input.getConnectedControllers`.
+    :param uint64 eAction: The analog action to stop momentum for. This can be obtained from :func:`input.getAnalogActionHandle`.
+    :returns: nothing
+    :SteamWorks: `StopAnalogActionMomentum <https://partner.steamgames.com/doc/api/isteaminput#StopAnalogActionMomentum>`_
+
+.. function:: input.triggerHapticPulse()
+    
+    :param uint64 inputHandle: The handle of the controller to affect. Obtained from :func:`input.getConnectedControllers`.
+    :param boolean rightController: If `false`, affects the left haptic touch pad; otherwise affects the right pad.
+    :param short usDurationMicroSec: Duration of the pulse, in microseconds (1/1,000,000th of a second)
+    :returns: nothing
+    :SteamWorks: `TriggerHapticPulse <https://partner.steamgames.com/doc/api/isteaminput#TriggerHapticPulse>`_
+
+.. function:: input.triggerRepeatedHapticPulse()
+    
+    :param uint64 inputHandle: The handle of the controller to affect. Obtained from :func:`input.getConnectedControllers`.
+    :param boolean rightController: If `false`, affects the left haptic touch pad; otherwise affects the right pad.
+    :param short usDurationMicroSec: Duration of the pulse, in microseconds (1/1,000,000th of a second)
+    :param short usOffMicroSec: Duration of the pause between pulses, in microseconds.
+    :param short unRepeat: 	Number of times to repeat the `usDurationMicroSec / usOffMicroSec` duty cycle.
+    :returns: nothing
+    :SteamWorks: `TriggerRepeatedHapticPulse <https://partner.steamgames.com/doc/api/isteaminput#TriggerRepeatedHapticPulse>`_
+
+.. function:: input.triggerVibration()
+    
+    :param uint64 inputHandle: The handle of the controller to affect. Obtained from :func:`input.getConnectedControllers`.
+    :param short usLeftSpeed: The intensity value for the left rumble motor.
+    :param short usRightSpeed: The intensity value for the right rumble motor.
+    :returns: nothing
+    :SteamWorks: `TriggerVibration <https://partner.steamgames.com/doc/api/isteaminput#TriggerVibration>`_
+
+.. function:: input.getActionOriginFromXboxOrigin()
+    
+    :param uint64 inputHandle: The handle of the controller to affect. Obtained from :func:`input.getConnectedControllers`, or from :func:`input.getControllerForGamepadIndex`.
+    :param string eOrigin: This is the button you want to get the image for example: A, B, LeftTrigger_Click, LeftStick_DPadWest.
```

Comment:
The description of the origin parameters are inconsistent. This one doesn't have link, others don't have examples.

I think it would be nice to be something like this on all of them:
"function_specific_param_description. Example: A, LeftTrigger_Click (see \<EInputActionOrigin\>(...) for all options)".